.github/workflows/release.yaml

```diff
@@ -5,18 +5,20 @@ on:  # yamllint disable-line rule:truthy
   workflow_call:
   workflow_dispatch:
     inputs:
-      version:
-        description: 'Version to release (major, minor, patch), Example: 1.x.x'
-        required: true
-      image:
-        description: 'Image name. Example: csi-powerstore'
-        default: 'csi-powerscale'
+      option:
+        description: 'Select version to release'
         required: true
+        type: choice
+        default: 'minor'
+        options:
+          - major
+          - minor
+          - patch
 jobs:
   csm-release:
     uses: dell/common-github-actions/.github/workflows/csm-release-driver-module.yaml@main
     name: Release CSM Drivers and Modules
     with:
-      version: ${{ github.event.inputs.version }}
-      image: ${{ github.event.inputs.image }}
+      version: ${{ github.event.inputs.option }}
+      images: 'csi-powerscale'
```

Comment:
shouldn't this be csi-isilon?

Reply:
Good catch. @harishp8889 Please check.
dellemc/csi-isilon:v2.13.0